chat-commands.js

```diff
@@ -565,19 +566,19 @@ exports.commands = {
 		if (!target) target = user.name;
 
 		Ladders.visualizeAll(target).then(values => {
-			let buffer = '<div class="ladder"><table>';
-			buffer += '<tr><td colspan="8">User: <strong>' + Chat.escapeHTML(target) + '</strong></td></tr>';
+			let buffer = `<div class="ladder"><table>`;
+			buffer += `<tr><td colspan="8">User: <strong>${Chat.escapeHTML(target)}</strong></td></tr>`;
 
-			let ratings = values.join('');
+			let ratings = values.join(``);
 			if (!ratings) {
-				buffer += '<tr><td colspan="8"><em>This user has not played any ladder games yet.</em></td></tr>';
+				buffer += `<tr><td colspan="8"><em>This user has not played any ladder games yet.</em></td></tr>`;
 			} else {
-				buffer += '<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th>W</th><th>L</th><th>Total</th>';
+				buffer += `<tr><th>Format</th><th><abbr title="Elo rating">Elo</abbr></th><th>W</th><th>L</th><th>Total</th>`;
 				buffer += ratings;
 			}
 			buffer += '</table></div>';
 
-			this.sendReply('|raw|' + buffer);
+			this.sendReplyBox(buffer);
```

Comment:
this should be left as ``this.sendReply(`|raw|${buffer}`);``

Reply:
Yeah, replyboxes are, like, boxes.